src/core/compact_object.h

```diff
@@ -135,7 +135,7 @@ class CompactObj {
     NONE_ENC = 0,
     ASCII1_ENC = 1,
     ASCII2_ENC = 2,
-    HUFFMAN_ENC = 3,  // TBD
+    HUFFMAN_ENC = 3,
```

Comment:
I had in mind that HUFFMAN_ENC splits into HUFFMAN_KEYS_ENC and HUFFMAN_VAL_ENC, so you don't need any new variables at all